deploy/template.yaml

```diff
@@ -104,6 +104,9 @@ Conditions:
   IsDev01: !Equals [ !Ref AWS::AccountId, "130355686670"]
   IsDev02: !Equals [ !Ref AWS::AccountId, "175872367215"]
   IsNotDevelopment: !Not [ !Condition IsDevelopment ]
+  IsDevOrBuild: !Or
+    - !Condition IsDevelopment
+    - !Equals [ !Ref Environment, build ]
```

Comment:
FWIW I think this might be slightly less brittle if it's named `HasInternalTestApi` or something, but it's not a big deal.

(In future you might change which environments the test api is deployed to, but you might not want to change other things conditional on 'dev or build')

Reply:
Yeah, I did think about that, and then didn't do it, for no really good reason. I'll fix it up.